api/controllers/project.js

```diff
@@ -365,7 +365,7 @@ exports.protectedDelete = function (args, res, next) {
 exports.protectedPost = function (args, res, next) {
   var obj = args.swagger.params.project.value;
 
-  // default project creation is set to 2002 right now for backwards compatibility with other apps tat use this api
+  // default project creation is set to 2002 right now for backwards compatibility with other apps that use this api
   var projectLegislationYear = obj.legislationYear ? obj.legislationYear : 2002;
```

Comment:
we will need to think about backwards compatibility for this case

Reply:
This is just making an assumption that is accurate about how we have put all our project data into the `legislation_2002` key. New version of the web app will pass through the legislation year